src/server/pkg/collection/collection.go

```diff
@@ -254,6 +255,10 @@ func (c *readWriteCollection) TTL(key string) (int64, error) {
 }
 
 func (c *readWriteCollection) PutTTL(key string, val proto.Message, ttl int64) error {
+	if strings.Contains(key, indexIdentifier) {
+		fmt.Printf("Illegal key: %s\n", key)
+		return fmt.Errorf("cannot put key %q which contains reserved string %q", key, indexIdentifier)
```

Comment:
Did this happen? How? (just curious)

Reply:
It kinda did, for a while the check was wrong and got triggered on the actual index entries, which is what caused me to add the debug line. It never got triggered on anything real, but I wanted to leave it in anyway just for paranoia sake.